src/events.rs

```diff
@@ -203,35 +167,44 @@ pub enum EventType {
     ///
     /// `chat_id` is set if only a single chat is affected by the changes, otherwise 0.
     /// `msg_id` is set if only a single message is affected by the changes, otherwise 0.
-    #[strum(props(id = "2000"))]
-    MsgsChanged { chat_id: ChatId, msg_id: MsgId },
+    MsgsChanged {
+        chat_id: ChatId,
+        msg_id: MsgId,
+    },
 
     /// There is a fresh message. Typically, the user will show an notification
     /// when receiving this message.
     ///
     /// There is no extra #DC_EVENT_MSGS_CHANGED event send together with this event.
-    #[strum(props(id = "2005"))]
-    IncomingMsg { chat_id: ChatId, msg_id: MsgId },
+    IncomingMsg {
+        chat_id: ChatId,
+        msg_id: MsgId,
+    },
```

Comment:
why this reformatting?

Reply:
It's `rustfmt` (`cargo fmt`), I don't format the code manually.